build.gradle.kts

```diff
@@ -23,6 +23,11 @@ repositories {
 }
 
 dependencies {
+    implementation("com.aries:docker-java-shaded:3.1.0-rc-7:cglib@jar") {
+        setTransitive(false)
+    }
+    implementation("org.slf4j:slf4j-simple:1.7.5")
```

Comment:
@cdancy Can you explain why we need the SLF4J and Activation libraries? Shouldn't those be included in the Docker Java library as shaded packages?